### ISSUE TITLE
WPS517: missed cases

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1759]
# Bug report

## What's wrong

"WPS517 | Found pointless starred expression" doesn't report string unpacking, `dict` constructor, and `list` conversion:

```python3
sum(**dict(a=2))
sum(*'hello')
sum(*list('hello'))
```
## System information

[flakehell.orsinium.dev](https://flakehell.orsinium.dev/) playground

**Comment by sobolevn** [comment]
I leave `*list()` and `**dict()` for the `typed-linter`

**Comment by AIGeneratedUsername** [comment]
I am sorry for bumping an old issue, but I really wonder about what this rule.

Here is minimalist code
```python
def func(foo, *, bar, **kwargs):
    print(foo)
    print(bar)
    print(kwargs)


request_headers = {"spam": get_spam(), "eggs": calculated_value}

func("value", bar="hello", **request_headers)
```
I do not understand why I am asked to pass `request_headers` instead of the `**request_headers` when I need to pass kwargs from my dictionary with some dynamically generated data.

This is so obvious use case so either I am missing something, or you overlooked such use case when designing this rule. Thanks.
